recipes/psfgen/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "psfgen" %}
+{% set version = "1.0.0" %}
+{% set sha256 = "6bd766ec837f0cd7c42cd110ae81acb5405134bb534bbc38cc10030c5613ab88" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    fn: {{ name }} -v{{ version }}.tar.gz
+    url: https://github.com/Eigenstate/{{ name }}/archive/v{{ version }}.tar.gz
+    sha256: {{ sha256 }}
+
+build:
+    number: 0
+    skip: true  # [win]
+    script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+    build:
+        - {{ compiler('c') }}
+    host:
+        - python
```

Comment:
Please, add ``setuptools``, it seems you need it as a dependency for your build system